dev/integration_tests/channels/macos/Runner/Configs/AppInfo.xcconfig

```diff
@@ -0,0 +1,14 @@
+// Application-level settings for the Runner target.
+//
+// This may be replaced with something auto-generated from metadata (e.g., pubspec.yaml) in the
+// future. If not, the values below would default to using the project name when this becomes a
+// 'flutter create' template.
+
+// The application's name. By default this is also the title of the Flutter window.
+PRODUCT_NAME = channels
+
+// The application's bundle identifier
+PRODUCT_BUNDLE_IDENTIFIER = com.yourcompany.channels
```

Comment:
This isn't right in the iOS channels project either, but the convention for the example projects is `dev.flutter`.
```suggestion
PRODUCT_BUNDLE_IDENTIFIER = dev.flutter.channels
```